src/cli/commands/force/lightning/local/device/list.ts

```diff
@@ -63,14 +63,16 @@ export default class List extends SfdxCommand {
     }
 
     public async iOSDeviceList(): Promise<IOSSimulatorDevice[]> {
+        const startTime = new Date().getTime();
         const result = await IOSUtils.getSupportedSimulators();
-        this.showDeviceList(result);
+        this.showDeviceList(result, startTime);
```

Comment:
I don't think it's the right approach to pollute the method signatures with parameters for performance data. Also, we should be standardizing on the [existing tools in Node.js](https://nodejs.org/docs/latest-v12.x/api/perf_hooks.html) to do these kinds of measurements.

Using an approach that sets named markers gives us the ability to share those markers in more of a global context, obviating the need to pass performance data into each method.

Reply:
hmmm... I was following the pattern that we have in the VSCode extension repo for [our code](https://github.com/forcedotcom/salesforcedx-vscode/blob/03ec61e01d5be51c2fddbdf3995b9e8991961a75/packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcPreview.ts#L163). I'll look into using the perf APIs of Node.js then.